.github/workflows/release.yml

```
@@ -75,45 +75,34 @@ jobs:
-          # Build the release
           mix release
 
-      - name: Debug Dockerfile
-        run: |
-          echo "Current directory: $(pwd)"
-          echo "Listing files in current directory:"
-          ls -la
-          echo "Dockerfile content:"
-          cat Dockerfile
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)